tests/test_aaai_conference.py

```diff
@@ -73,7 +73,9 @@ def test_create_conference(self, client, openreview_client, helpers, profile_man
                 'Expected Submissions': '10000',
                 'use_recruitment_template': 'Yes',
                 'api_version': '2',
-                'submission_license': ['CC BY 4.0']
+                'submission_license': ['CC BY 4.0'],
+                'iThenticate_plagiarism_check': 'Yes',
+                'iThenticate_plagiarism_check_api_key': '1234',
```

Comment:
@akshatchhabra add to the test the value for `iThenticate_plagiarism_check_api_base_url`

Reply:
Done